### ISSUE TITLE
No distinction between removing and deleting a Set

### ISSUE THREAD

**Issue by darmagan** [Icinga/icingaweb2-module-director#1107]
If I remove a service set from a Host, the log entry says it's been deleted. If I delete it completely, it says that it's deleted, too. 

## Expected Behavior
Distinction between these two operations in the Logs.

## Current Behavior
No distinction. It's not clear if  the object was deleted or not from the Activity Log

## Steps to Reproduce (for bugs)
1. create set
2. apply to host
3. remove from host
4. delete it afterwards
5. examine the logs

![auswahl_018](https://user-images.githubusercontent.com/25243900/29673338-3b6043c4-88ef-11e7-9828-d1cf1b10bb97.png)

from those entries, it looks like the set was deleted two times. Now was it just removed from hosts or was it deleted? Bad for debugging purposes.

## Your Environment
root@box18:/usr/share/icingaweb2/modules/director# git log -n 1
commit c4a97692df23e428a2eb6f1be41a7c25ea7b19a4
Author: Thomas Gelf <thomas@gelf.net>
Date:   Thu Jul 20 18:12:43 2017 +0200

    module.info: prepare v1.3.2
root@box18:/usr/share/icingaweb2/modules/director# git branch
* (detached from v1.3.2)
  master

Regards

**Comment by Thomas-Gelf** [comment]
@darmagan: well... it is deleted in any case, as every Set on every Host is a distinct object inheriting from the "real" Set (being a template). But you're right, this is confusing - we should somehow bring the host name into the log-key of the deleted object here.

**Comment by darmagan** [comment]
But there is a difference, if I remove the set from the host I delete the object, right. But if I  delete this:
![auswahl_019](https://user-images.githubusercontent.com/25243900/29674264-efacae56-88f1-11e7-8c7a-6f6e37de774a.png)

I am deleting the template. So if I look into the log I don't know if the template(the 'real' Set) was deleted or just the inherited object. And that's crucial for debugging (for example to reconstruct the steps taken)

**Comment by Thomas-Gelf** [comment]
Yes and no, you are once deleting a ServiceSet template and once a ServiceSet object. Every set attached to a host is internally being stored as a real ServiceSet object inheriting from the ServiceSet template.

**Comment by Thomas-Gelf** [comment]
@darmagan: regardless of those details, I'm completely with you - as I wrote above:

> But you're right, this is confusing - we should somehow bring the host name into the log-key of the deleted object here.

Will try to find a solution here.
